Topic recognition-OAI AKOA knee data set-ChenXu -s4547456

1154761334 opened the pull request.

1154761334 committed:
Create file

1154761334 committed:
Finish data process

1154761334 committed:
Input data

with ignore

1154761334 committed:
Create report.ipynb

1154761334 committed:
Finish model building and report

1154761334 committed:
finish report

SiyuLiu0329 commented:
- Commit messages should be a lot more specific
- Did not follow prescribed project structure. Submission crashes browser and corrupts repo.
- Code appears okay for Easy Difficulty 
- Performance appears okay

Other Feedback
- use .gitignore to exclude .DS_Store

rexsw commented:
Code looks good for easy difficulty

For next time;

Your pull request should have a description of what it is and why it should be merged 

Paths such as '/content/drive/My Drive/colab/P4/AKOA_Analysis.zip' won’t work for the user and should either be set dynamical or it needs to be made clear to the user that they need to change these paths

Code in files like driver scripts should be in a main function and in a python file.

Code needs more meaningful comments

Rember to remove unneeded commented out code

shakes76 commented:
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Feedback required. Revert Changes to my InfoVae project and Readme.**
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

1154761334 committed:
Fix mistake from  Feedback

comments:

Added new comments where the path needs to be changed
Make it clear to users that they need to change these paths

Newly created two folders jupyter notebook and code
Stored code and visual report separately

Removed unnecessary code

1154761334 committed:
Delete Conflicting files

1154761334 committed:
Delete Conflicting files

1154761334 committed:
Delete .DS_Store

Delete Conflicting files

1154761334 committed:
Delete .DS_Store

Conflicting files

1154761334 committed:
Delete .DS_Store

1154761334 committed:
Delete .DS_Store

shakes76 commented:
Still has conflict from .gitignore file

1154761334 committed:
Try fix confliect

1154761334 committed:
Merge branch 'topic-recognition' of https://github.com/1154761334/PatternFlow into topic-recognition

1154761334 committed:
try fix Conflicting

1154761334 committed:
fixing

1154761334 commented:
Thanks for all of you and your feedback.
After two days I finally resolve the Conflicting by manual operation :)

shakes76 commented:
You'll get the feedback marks but the PR is broken, there are 272 files now being changes from the entire repo. You will need to fix this if you want your algorithm merged with the main repo.

1154761334 commented:
Thanks，I will try to fix it.





------------------ Original ------------------
From: Shakes <notifications@github.com&gt;
Date: Wed, Nov 25, 2020 9:11 PM
To: shakes76/PatternFlow <PatternFlow@noreply.github.com&gt;
Cc: 1154761334 <1154761334@qq.com&gt;, Author <author@noreply.github.com&gt;
Subject: Re: [shakes76/PatternFlow] Topic recognition-OAI AKOA knee data set-ChenXu -s4547456 (#150)






You'll get the feedback marks but the PR is broken, there are 272 files now being changes from the entire repo. You will need to fix this if you want your algorithm merged with the main repo.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
